frontend/src/App.js

```diff
@@ -10,15 +10,19 @@ import InDepthDetailView from './components/inDepthDetailView/InDepthDetailView'
 import NewPlacePage from './components/NewPlacePage';
 import PlacesContextProvider from './contexts/PlacesContextProvider';
 import EditPlacePage from './components/EditPlacePage';
+import OverViewMaps from './components/OverViewMaps';
 
 function App() {
   return (
     <Router>
       <PlacesContextProvider>
         <Switch>
-          <Route path={'/overview'}>
+          <Route path={'/overviewlist'}>
             <OverViewList />
           </Route>
+          <Route path={'/overviewmaps'}>
+            <OverViewMaps />
+          </Route>
```

Comment:
What do you think about this?
```
<Route path="/overview/list" component={OverViewList} />
<Route path="/overview/maps" component={OverViewMap} />
```

would be consistent with the names of your other routes.